### DISCUSSION TITLE
CIT Resewn compatibility?

### DISCUSSION THREAD

**Discussion by Starexify** [Sinytra/ForgifiedFabricAPI#38] [Ideas]
Will there ever be a CIT Resewn compatiblitity with this mod?

**Comment by Su5eD** [comment]
Is it not compatible?

**Reply by Starexify** [reply]
I tried to use two resourcepacks that requires CIT Resewn or Optifine, and it doesnt work with Connector sadly. Even tho it loads on Connector.

**Reply by Su5eD** [reply]
Then it should be reported on Connector's repository instead of FFAPI.

**Reply by Starexify** [reply]
Oh sorry. Ill close this then
